packages/form-js-editor/test/spec/properties-panel/PropertiesPanel.spec.js

```diff
@@ -575,6 +580,108 @@ describe('properties panel', function() {
 
       });
 
+
+      describe('dynamic values', function() {
+
+        it('source should be configurable', function() {
+
+          // given
+          const editFieldSpy = spy();
+
+          const field = schema.components.find(({ key }) => key === 'product');
+
+          createPropertiesPanel({
+            container,
+            editField: editFieldSpy,
+            field
+          });
+
+          // assume
+          const input = screen.getByLabelText('Values source');
+
+          expect(input.value).to.equal('static');
+
+          // when
+          fireEvent.input(input, { target: { value: 'input' } });

```

Comment:
We could use `VALUES_SOURCES.INPUT` here, as I also wondered where the `input` value is coming from.